src/public/parts/shells/chat/public/group-ui.css

```
@@ -29,8 +29,8 @@
 	bottom: -72px;
 	width: 180px;
 	height: 180px;
-	border: 1px solid color-mix(in srgb, currentColor 22%, transparent);
-	border-radius: 50%;
+	border: var(--border) solid color-mix(in srgb, currentColor 22%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)